clevertap-core/src/test/java/com/clevertap/android/sdk/CleverTapAPITest.kt

```diff
@@ -586,23 +446,139 @@ class CleverTapAPITest : BaseTestCase() {
         corestate.controllerManager = controllerManager
 
         // Act
-        mockStatic(CTExecutorFactory::class.java).use {
-            `when`(CTExecutorFactory.executors(any())).thenReturn(
-                MockCTExecutors(
-                    cleverTapInstanceConfig
-                )
-            )
-            mockStatic(CleverTapFactory::class.java).use {
-                `when`(CleverTapFactory.getCoreState(application, cleverTapInstanceConfig, null))
-                    .thenReturn(corestate)
-                `when`(controllerManager.ctInboxController).thenReturn(inboxController)
-                cleverTapAPI = CleverTapAPI.instanceWithConfig(application, cleverTapInstanceConfig)
-                cleverTapAPI.markReadInboxMessagesForIDs(messageIDs)
+        setupBasicTest {
+            `when`(controllerManager.ctInboxController).thenReturn(inboxController)
+            initializeCleverTapAPI()
+            cleverTapAPI.markReadInboxMessagesForIDs(messageIDs)
+
+            // Assert
+            verify(controllerManager, times(2)).ctInboxController
+            verify(inboxController).markReadInboxMessagesForIDs(messageIDs)
+        }
+    }
 
-                // Assert
-                verify(controllerManager,times(2)).ctInboxController
-                verify(inboxController).markReadInboxMessagesForIDs(messageIDs)
-            }
+    @Test
+    fun `test getUserEventLogCount`(){
+        // Arrange
+        val evt = "test"
+        `when`(corestate.localDataStore.readUserEventLogCount(evt)).thenReturn(1)
+        
+        // Act
+        setupBasicTest {
+            initializeCleverTapAPI()
+            val userEventLogCountActual = cleverTapAPI.getUserEventLogCount(evt)
+
+            // Assert
+            assertEquals(1, userEventLogCountActual)
+            verify(corestate.localDataStore).readUserEventLogCount(evt)
+        }
+    }
+
+    @Test
+    fun `test getUserEventLog`(){
+        // Arrange
+        val evt = "test"
+        val log = UserEventLog(evt,1000L,1000L,1,"dId")
```

Comment:
There are already a lot of changes to this file, you can format the whole thing.

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/5a2f2c022aac769fc693f5ff6400bc6bfde61628